### ISSUE TITLE
New Localizations

### ISSUE THREAD

**Issue by Lattyware** [Lattyware/massivedecks#172]
This project welcomes localizations from the community. If you speak another language and can translate the application, we'd love to have help making Massive Decks easier for more people to play.

Please see [the guide to translation on the wiki](https://github.com/Lattyware/massivedecks/wiki/Translation) for how to get started, and please feel free to make an issue if you need additional help.

**Comment by fadila-amin** [comment]
Hi, i am a indonesian native. May i try to make an indonesian translation?

**Comment by Lattyware** [comment]
> Hi, i am a indonesian native. May i try to make an indonesian translation?

That would be great, as far as I know no one else is working on one right now, so feel free!

If you run into any difficulties, feel free to make an issue and I'll help when I get the chance.

**Comment by TheChilliPL** [comment]
There are some tiny issues in the Polish translation I noticed. They're really tiny and quick to fix so I don't even bother creating a separate issue: there are missing spaces after colons on lines 1212 and 1289 in the current master branch

**Comment by TheChilliPL** [comment]
Also, Indonesian in Polish is `Indonezyski`, I presume the translator messed that up, as it sometimes does

**Comment by CookieCoder15** [comment]
Maybe we could create a wiki page with missing translations and other translation issues such as the ones outlined in #135 and #178 to make it easier for translators to locate what is needed.

**Comment by TheChilliPL** [comment]
> Maybe we could create a wiki page with missing translations and other translation issues such as the ones outlined in #135 and #178 to make it easier for translators to locate what is needed.

I think it's a great idea!

**Comment by Lattyware** [comment]
> Maybe we could create a wiki page with missing translations and other translation issues such as the ones outlined in #135 and #178 to make it easier for translators to locate what is needed.

So I actually had [a GitHub action](https://github.com/Lattyware/auto-localization-issues) that I was working on to automatically find and add/update issues for missing strings in translations, so it didn't have to be managed manually, but there was some issue with it and I never got around to getting it working properly.

**Comment by TheChilliPL** [comment]
Would translating the original Cards Against Humanity deck to Polish be okay?

**Comment by Lattyware** [comment]
> Would translating the original Cards Against Humanity deck to Polish be okay?

Of course, you can see an example of it with [the pt-BR translation](https://github.com/Lattyware/massivedecks/blob/master/server/decks/cah-base-ptbr.deck.json5).

When it comes to the decks, there is much more room for localization over translation: the official CaH localized decks do things like swapping out America-centric references and jokes for ones that work better in the language. Obviously there is a huge amount of subjectivity there, so it is really up to you how far you want to go down that route, obviously I'm not in much of a position to say what makes sense, but obviously a 1:1 translation is also welcome, if you think that makes sense.

**Comment by TheChilliPL** [comment]
Reserving Spanish translation btw.

**Comment by TheChilliPL** [comment]
I translated most of it. There are plenty of `//TODO`-s I placed if I have an idea how could I translate the thing differently (and I'm not sure which to choose) or if I could swap it for some local reference.
It should be close to getting finished

**Comment by ucusita** [comment]
@Lattyware I can help you in spanish language.

**Comment by Unkn0wnCat** [comment]
Hey @Lattyware,

I'd love to rework the German translations, but I've encountered a problem.

In German words behave different depending on their position in the sentence.
For example:
`I will read the *black card*` => `Ich werde die *schwarze Karte* lesen`
`I will read the sentence from the *black card*` => `Ich werde den Satz von der *schwarzen Karte* lesen`
Notice how in the second sentence the word "schwarze" has an additional "n" at the end. This is due to it being used as the so called "Dativ" instead of the "Nominativ".

Is there a way to express this in the translation?

**Comment by Lattyware** [comment]
@Unkn0wnCat There are two ways you can approach this, one is just to replace references with absolute text, this is simpler but increases maintenance burden as changing one doesn't change the rest. The alternative is to provide enough context to use the correct version, if you want an example of that, take a look at [the Polish translation](https://github.com/Lattyware/massivedecks/blob/main/client/src/elm/MassiveDecks/Strings/Languages/Pl.elm) which does this.

**Comment by Unkn0wnCat** [comment]
@Lattyware The Polish translation is actually doing the same thing the German one needs to do. I'll try to get this to work for the German translation too! Thanks!

**Comment by TheChilliPL** [comment]
@Unkn0wnCat I was the translator for the Polish version, let me know if you need any help!

**Comment by rafitamolin** [comment]
Hey @Lattyware! I have submitted a PR to update the Spanish language.

I discovered Massive Decks yesterday, and after playing it for several hours with friends, I had an incredibly fun night. As a thank you, I have perfected the Spanish language so that the Hispanic community can enjoy the game as much as possible.

Thank you for this open source work of art! 🤗
